hooks/useHotWalletPluginUXDStaking.ts

```diff
@@ -60,48 +65,54 @@ const useHotWalletPluginUXDStaking = (hotWalletAccount: HotWalletAccount) => {
         );
       }
 
+      const sssClient = SingleSideStakingClient.load({
+        provider: new SolanaAugmentedProvider(
```

Comment:
Didn't we remove that from the consumer & hide it in the client instead?

Reply:
Yes the governance have not updated to the latest client yet, will do

Reply:
Done :)